### PR TITLE
ntfy: add page

### PR DESCRIPTION
**Pull request by CovertDuck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): 2.11.0**

### CODE REVIEW

File: pages/common/ntfy.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# ntfy
+
+> Send and receive POST notifications from the command line.
```
Comment (spageektti) [inline pages/common/ntfy.md:3]:
```suggestion
> Send and receive POST notifications.
```
See the style guide.

**Comment by spageektti** [comment]
Hi. Thanks for your contribution.

**Comment by CovertDuck** [comment]
No matter how many `tldr` contributions I make... I keep forgetting about a thing or two. Regardless, thanks for the help @sebastiaanspeck! :)

File: pages/common/ntfy.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# ntfy
+
+> Send and receive POST notifications.
```
Comment (acuteenvy) [inline pages/common/ntfy.md:3]:
```suggestion
> Send and receive HTTP POST notifications.
```

File: pages/common/ntfy.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://github.com/binwiederhier/ntfy>.
+
+- Send a message to the `security` topic:
+
+`ntfy pub security "{{Front door has been opened.}}"`
+
+- Send with title, priority and tags:
```
Comment (acuteenvy) [inline pages/common/ntfy.md:10]:
```suggestion
- Send with a title, priority and tags:
```

**Review by acuteenvy — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**